app/styles/ember-frost-table/_frost-table.scss

```diff
@@ -30,7 +49,19 @@
     }
   }
 
+  &-body {
```

Comment:
I believe we still want the default for the table body to be fixed - it's more performant than auto and flex is entirely different